lib/pure/includes/osseps.nim

```diff
@@ -1,66 +1,95 @@
 # Include file that implements 'DirSep' and friends. Do not import this when
 # you also import ``os.nim``!
 
+# Improved based on info in 'compiler/platform.nim'
+
 const
   doslikeFileSystem* = defined(windows) or defined(OS2) or defined(DOS)
 
-when defined(Nimdoc): # only for proper documentation:
-  const
-    CurDir* = '.'
-      ## The constant character used by the operating system to refer to the
-      ## current directory.
-      ##
-      ## For example: `'.'` for POSIX or `':'` for the classic Macintosh.
+const
+  CurDir* =
+    when defined(macos): ':'
+    elif defined(genode): '/'
```

Comment:
@araq @genotrance @ehmry is `/` (instead of `.`) for indicating current dir correct for genode ? couldn't find any info on the web regarding that

Reply:
Yes, I was the one how set it to `/`. I think the reasoning was that without a libc the current working directory can only be `/`.